src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const BASE_URL = 'https://mate.academy/students-ap';
```

Comment:
```suggestion
const BASE_URL = 'https://mate.academy/students-api';
```
it is reason why occurs error during fetch todos
![Screenshot from 2023-11-14 16-01-24](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/33029864/c1b6ce87-1fde-4be5-b86c-b1a9040b1886)
![Screenshot from 2023-11-14 16-01-17](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/33029864/b41451c2-6875-4ada-9a53-5dacf608809c)